src/daemon/rpmostree-sysroot-upgrader.cxx

```diff
@@ -433,13 +433,28 @@ rpmostree_sysroot_upgrader_pull_base (RpmOstreeSysrootUpgrader *self, const char
         if (override_commit)
           return glnx_throw (error, "Specifying commit overrides for container-image-reference "
                                     "type refspecs is not supported");
-        if (check)
-          return glnx_throw (error, "Cannot currently check for updates without downloading");
 
         CXX_TRY_VAR (import,
                      rpmostreecxx::pull_container (*self->repo, *cancellable, r.refspec.c_str ()),
```

Comment:
The hard part is that this needs to change...the idea behind "check" is that we fetch just the metadata, not all of the data.  

The use case here is e.g. supporting a user interface showing something like "update available, 1.3 GB download".

For container images, this means just fetching the manifest and not the layers; but we also want to compute the size of the layers we would fetch.

The necessary logic is already in the ostree-ext Rust code, but would need to be plumbed through here too.